app/models/framework/definition.rb

```diff
@@ -5,16 +5,11 @@ module Definition
     class MissingError < StandardError; end
 
     class << self
-      def cache
-        @cache ||=
-          Hash.new do |hash, framework_short_name|
-            hash[framework_short_name] = Framework::Definition::Language[framework_short_name]
-          rescue ArgumentError
-            raise Framework::Definition::MissingError, %(There is no framework definition for "#{framework_short_name}")
-          end
+      def [](framework_short_name)
+        Framework::Definition::Language[framework_short_name]
```

Comment:
These changes are making me wonder if the `Language` class has now been hollowed out to the point the it's almost useless, and its job could be rolled into the `Definition` class...

Reply:
I think you're right. I'll add a refactor commit for it.

Reply:
Done.